client/client-lib/src/api.rs

```diff
@@ -256,7 +256,7 @@ impl<C: JsonRpcClient + Send + Sync> FederationApi for WsFederationApi<C> {
         self.request(
             "/ln/register_gateway",
             gateway,
-            CurrentConsensus::new(self.max_evil),
+            CurrentConsensus::new(self.max_evil * 2),
```

Comment:
Since we fetch gateways from `f+1` members we need to register with `2f+1` members to ensure we always can retrieve a registered gateway.

Reply:
I suspect this might also be related to other failures e.g. in `CLI Test`